specification/ml/_types/Job.ts

```diff
@@ -59,63 +59,273 @@ export class JobStatistics {
+   * Valid values range from 0 to `model_snapshot_retention_days`.
+   * @server_default 1
+   */
   daily_model_snapshot_retention_after_days?: long
+  /**
+   * The data description defines the format of the input data when you send data to the job by using the post data API.
+   * Note that when configure a datafeed, these properties are automatically set.
+   * When data is received via the post data API, it is not stored in Elasticsearch.
+   * Only the results for anomaly detection are retained.
+   */
   data_description: DataDescription
+  /**
+   * The datafeed, which retrieves data from Elasticsearch for analysis by the job.
+   * You can associate only one datafeed with each anomaly detection job.
+   */
   datafeed_config?: Datafeed
+  /**
+   * Indicates that the process of deleting the job is in progress but not yet completed.
+   * It is only reported when `true`.
+   */
   deleting?: boolean
+  /**
+   * A description of the job.
+   */
   description?: string
+  /**
+   * If the job closed or failed, this is the time the job finished, otherwise it is `null`.
+   * This property is informational; you cannot change its value.
+   */
   finished_time?: DateTime
+  /**
+   * A list of job groups.
+   * A job can belong to no groups or many.
+   */
   groups?: string[]
+  /**
+   * Identifier for the anomaly detection job.
+   * This identifier can contain lowercase alphanumeric characters (a-z and 0-9), hyphens, and underscores.
+   * It must start and end with alphanumeric characters.
+   */
   job_id: Id
+  /**
+   * Reserved for future use, currently set to `anomaly_detector`.
+   */
   job_type?: string
+  /**
+   * The machine learning configuration version number at which the the job was created.
+   */
   job_version?: VersionString
+  /**
+   * This advanced configuration option stores model information along with the results.
+   * It provides a more detailed view into anomaly detection.
+   * Model plot provides a simplified and indicative view of the model and its bounds.
+   */
   model_plot_config?: ModelPlotConfig
   model_snapshot_id?: Id
+  /**
+   * Advanced configuration option, which affects the automatic removal of old model snapshots for this job.
+   * It specifies the maximum period of time (in days) that snapshots are retained.
+   * This period is relative to the timestamp of the most recent snapshot for this job.
+   * The default value is `10`, which means snapshots ten days older than the newest snapshot are deleted.
+   */
   model_snapshot_retention_days: long
+  /**
+   * Advanced configuration option.
+   * The period over which adjustments to the score are applied, as new data is seen.
+   * The default value is the longer of 30 days or 100 `bucket_spans`.
+   */
   renormalization_window_days?: long
+  /**
+   * A text string that affects the name of the machine learning results index.
+   * The default value is `shared`, which generates an index named `.ml-anomalies-shared`.
```

Comment:
```suggestion
   * The default value is `shared`, which generates an index named `.ml-anomalies-shared`.
   * @server_default shared
```

Reply:
I intentionally left the default values out here as the generator complains if I add a default value for an optional parameter.